pkg/controllers/reallocation/utilization.go

```diff
@@ -109,12 +112,29 @@ func (u *Utilization) terminateExpired(ctx context.Context, provisioner *v1alpha
 	if err != nil {
 		return fmt.Errorf("listing underutilized nodes, %w", err)
 	}
-
 	// 2. Trigger termination workflow if past TTLAfterEmpty
 	for _, node := range nodes {
 		if utilsnode.IsPastEmptyTTL(node) {
 			logging.FromContext(ctx).Infof("Triggering termination for empty node %s", node.Name)
-			if err := u.kubeClient.Delete(ctx, node); err != nil {
+			if err := u.KubeClient.Delete(ctx, node); err != nil {
+				return fmt.Errorf("sending delete for node %s, %w", node.Name, err)
```

Comment:
consider `deleting node`. Sending delete for is a bit of a nod and wink to finalizers, which we know may or may not exist for all kubernetes objects. I'd hate to be forced into "sending" language everywhere because of this. Easier to just use the rest concept `delete`, regardless of finalization hooks.